tailwind.config.js

```diff
@@ -1,11 +1,40 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: [
-        "./src/app/**/*.{html,js}",
-    ],
+  content: ["./src/**/*.{html,js}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        primary: "#EAF8F6",
+        secondary: "#0D6C5F",
+        white: "#ffffff",
+        balanpy: {
+          50: "#EAF8F6",
+          100: "#D5F4F0",
+          200: "#C6EBE6",
+          300: "#B1E5DE",
+          400: "#9FE2D9",
+          500: "#87D5CA",
+          DEFAULT: "#70CABD",
```

Comment:
Tío, tienes que mantener el 600 también, si no cómo adivina tailwind qué quieres decir si pones `text-balanpy-600`?
```suggestion
          600: "#70CABD",
          DEFAULT: "#70CABD",
```